src/events/event_publisher.cpp

```
@@ -74,17 +72,17 @@ class EventPublisherImpl : public transport::ITransportListener {
         stop_publish_timer();
 
         // Clear all subscriptions and events
-        std::scoped_lock subs_lock(subscriptions_mutex_);
+        platform::ScopedLock subs_lock(subscriptions_mutex_);
         subscriptions_.clear();
 
```

Cppcheck (2.19.0)

[style] The function 'someip_set_nonblocking' is never used.

(unusedFunction)